examples/all_in_one/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Not sure if we need this script at all? It could be started up just using `docker-compose` without hiding anything from the user.

Reply:
I've added the script as an entrypoint to a new service within docker-compose.

https://github.com/seznam/slo-exporter/pull/4/commits/f55a356970cf3dfdda1a15c0e774a8336a20e028